src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
I'm still learning Roslyn here so any feedback and advice is welcome.
Do I have to explicitly call the Simplifier? I didn't have any references before, so I've added the reference to mscorlib to solve the CommentID issue I was having before. But I could still be missing tons of other needed references.